Documentation incorrectly specifies that local functions cannot be static

I believe this line is now outdated, since local functions can now be static:

"Including the static keyword generates compiler error CS0106, 'The modifier 'static' is not valid for this item.'".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c0a70d19-0d23-f877-8860-df514cec3c40
* Version Independent ID: 942dbec3-a753-733f-c551-03d203e48969
* Content: [Local functions - C# Programming Guide](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/local-functions#feedback)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/local-functions.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/local-functions.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -37,12 +37,10 @@ Local functions can use the [async](../../language-reference/keywords/async.md) 
 
 Note that all local variables that are defined in the containing member, including its method parameters, are accessible in the local function. 
 
-Unlike a method definition, a local function definition cannot include the following elements:
+Unlike a method definition, a local function definition cannot include the member access modifier. Because all local functions are private, including an access modifier, such as the `private` keyword, generates compiler error CS0106, "The modifier 'private' is not valid for this item."
 
-- The member access modifier. Because all local functions are private, including an access modifier, such as the `private` keyword, generates compiler error CS0106, "The modifier 'private' is not valid for this item."
-
-  > [!NOTE]
-  > Prior to C# 8.0, local functions cannot include the `static` modifier. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
+> [!NOTE]
+> Prior to C# 8.0, local functions cannot include the `static` modifier. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
 In addition, attributes can't be applied to the local function or to its parameters and type parameters. 
  

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -44,7 +44,7 @@ Unlike a method definition, a local function definition cannot include the follo
 - The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
   > [!NOTE]
-  > Starting with C# 8.0, `static` local functions are valid. A `static` local function cannot access variables from the containing method or instance fields.
+  > Prior to C# 8.0, local functions cannot include the `static` modifier. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
 In addition, attributes can't be applied to the local function or to its parameters and type parameters. 
  

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -40,8 +40,11 @@ Note that all local variables that are defined in the containing member, includi
 Unlike a method definition, a local function definition cannot include the following elements:
 
 - The member access modifier. Because all local functions are private, including an access modifier, such as the `private` keyword, generates compiler error CS0106, "The modifier 'private' is not valid for this item."
- 
+
 - The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
+
+  > [!NOTE]
+  > Starting from C# 8, `static` local functions are valid. A `static` local function wouldn't be able to access variables from  parent method or instance fields.
 
 In addition, attributes can't be applied to the local function or to its parameters and type parameters. 
  

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -41,8 +41,6 @@ Unlike a method definition, a local function definition cannot include the follo
 
 - The member access modifier. Because all local functions are private, including an access modifier, such as the `private` keyword, generates compiler error CS0106, "The modifier 'private' is not valid for this item."
 
-- The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
-
   > [!NOTE]
   > Prior to C# 8.0, local functions cannot include the `static` modifier. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -44,7 +44,7 @@ Unlike a method definition, a local function definition cannot include the follo
 - The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
   > [!NOTE]
-  > Starting with C# 8, `static` local functions are valid. A `static` local function wouldn't be able to access variables from parent method or instance fields.
+  > Starting with C# 8.0, `static` local functions are valid. A `static` local function cannot access variables from the containing method or instance fields.
 
 In addition, attributes can't be applied to the local function or to its parameters and type parameters. 
  

--- a/docs/csharp/programming-guide/classes-and-structs/local-functions.md
+++ b/docs/csharp/programming-guide/classes-and-structs/local-functions.md
@@ -44,7 +44,7 @@ Unlike a method definition, a local function definition cannot include the follo
 - The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
   > [!NOTE]
-  > Starting from C# 8, `static` local functions are valid. A `static` local function wouldn't be able to access variables from  parent method or instance fields.
+  > Starting with C# 8, `static` local functions are valid. A `static` local function wouldn't be able to access variables from parent method or instance fields.
 
 In addition, attributes can't be applied to the local function or to its parameters and type parameters. 
  

